### PR TITLE
3924 schema

### PR DESCRIPTION
**Pull request by naveen-gowda-arista**

### CODE REVIEW

File: v1/quota_manager/__pycache__/validate_quota_manager.cpython-39.pyc
Comment (Utkarsh51) [inline v1/quota_manager/__pycache__/validate_quota_manager.cpython-39.pyc:1]: I don;t think this file should be a prt of the PR

File: .vscode/tasks.json [hunk 1]
```diff
@@ -47,8 +47,18 @@
                 "kind": "test",
                 "isDefault": true
             }
+        },
+	    {
+            "label": "Test Quota Manager",
+            "type": "shell",
+            "command": "./validate.py Quota Manager v1/quota_manager/test_schema.json v1/quota_manager/quota_manager_test.json",
```
Comment (Utkarsh51) [inline .vscode/tasks.json:54]: the argument Quota Manager should be the key defined validate.py

**Review by Utkarsh51 — CHANGES_REQUESTED** [review]

File: .vscode/tasks.json [hunk 2]
```diff
@@ -47,8 +47,18 @@
...
+            "group": {
+                "kind": "test",
+                "isDefault": true
+            }
         }
-    ],
```
Comment (Utkarsh51) [inline .vscode/tasks.json:51]: Why the comma is removed?

**Review by Utkarsh51 — CHANGES_REQUESTED** [review]

File: v1/quota_manager/__pycache__/validate_quota_manager.cpython-39.pyc
Comment (naveen-gowda-arista) [reply v1/quota_manager/__pycache__/validate_quota_manager.cpython-39.pyc:1]: Ok. Removed it.

File: .vscode/tasks.json (hunk 2 above)
Comment (naveen-gowda-arista) [reply .vscode/tasks.json:51]: Looks like typo. Fixed it.

File: .vscode/tasks.json (hunk 1 above)
Comment (naveen-gowda-arista) [reply .vscode/tasks.json]: Fixed it.

File: .vscode/tasks.json [hunk 3]
```diff
@@ -47,7 +47,17 @@
                 "kind": "test",
                 "isDefault": true
             }
+        },
+	    {
+            "label": "Test Quota Manager",
+            "type": "shell",
+            "command": "./validate.py quota_manage v1/quota_manager/test_schema.json v1/quota_manager/quota_manager_test.json",
```
Comment (Utkarsh51) [inline .vscode/tasks.json:54]: this still would give an error "quota_manage" doesn't match with what is defined in validate.py

**Review by Utkarsh51 — CHANGES_REQUESTED** [review]

File: .vscode/tasks.json (hunk 3 above)
Comment (naveen-gowda-arista) [reply .vscode/tasks.json]: I hace fixed and ran the same command from .vscode and ran the test...here is the output.
[Is there any other way to test this apart from this method]


~/untangle/mfw_schema @us277.sjc> ./validate.py quota_manager v1/quota_manager/test_schema.json v1/quota_manager/quota_manager_test.json
=== START OF MODULE TEST: <module 'v1.quota_manager.validate_quota_manager' from '/home/naveen.gowda/untangle/mfw_schema/v1/quota_manager/validate_quota_manager.py'> ===

test_base_id_uniqueness (v1.quota_manager.validate_quota_manager.TestQuotaManager)
Tests that the base id's (e.g., quota/id or configuration/id) are all only used once at that level. That ... ok
test_configuration_ids_in_quota_rules (v1.quota_manager.validate_quota_manager.TestQuotaManager)
Tests that any configuration id's under quota rule's actions (but not the other way around) ... ok
test_exceed_action_ids_in_rules (v1.quota_manager.validate_quota_manager.TestQuotaManager)
Tests that any exceed action id's under rules exist in exceed_actions (but not the other way around) ... ok
test_valid_ids (v1.quota_manager.validate_quota_manager.TestQuotaManager)
Testing that each quota_manager lists configuration's id is a valid uuid ... ok
{'configurations': {'orphans': ['1c60f45e-919b-11ee-b9d1-0242ac120002']}, 'exceed_actions': {'orphans': ['5120fbfc-919a-11ee-b9d1-0242ac120002']}, 'rules': {}, 'rule_conditions': {}, 'rule_action': {}}

----------------------------------------------------------------------
Ran 4 tests in 0.009s

OK
=== END OF MODULE TEST: <module 'v1.quota_manager.validate_quota_manager' from '/home/naveen.gowda/untangle/mfw_schema/v1/quota_manager/validate_quota_manager.py'> ===

~/untangle/mfw_schema @us277.sjc>

**Review by Utkarsh51 — APPROVED** [review]
LGTM

**Pull request merged**